internal/bridge/crunchybridgecluster/crunchybridgecluster_controller.go

```diff
@@ -600,8 +513,8 @@ func (r *CrunchyBridgeClusterReconciler) GetSecretKeys(
 			return string(existing.Data["key"]), string(existing.Data["team"]), nil
 		}
 		err = fmt.Errorf("error handling secret: found key %t, found team %t",
```

Comment:
It looks like this error message would be
> error handling secret: found key (true or false), found team (true or false)

I can't quite tell what is going on with this error message. Maybe say what we are expecting to find? :pick: 

something like ?
> expected to find a key and a team: found key true, found team false